Tests crash with a SIGSEGV

### Description
When attempting to run the tests they crash with a SIGSEGV:
```
go test -cover -timeout 30s -race github.com/eiffel-community/eiffel-goer/cmd/goer github.com/eiffel-community/eiffel-goer/internal/config github.com/eiffel-community/eiffel-goer/internal/database github.com/eiffel-community/eiffel-goer/internal/database/drivers github.com/eiffel-community/eiffel-goer/internal/database/drivers/mongodb github.com/eiffel-community/eiffel-goer/internal/logger github.com/eiffel-community/eiffel-goer/internal/query github.com/eiffel-community/eiffel-goer/internal/requests github.com/eiffel-community/eiffel-goer/internal/responses github.com/eiffel-community/eiffel-goer/internal/schema github.com/eiffel-community/eiffel-goer/pkg/application github.com/eiffel-community/eiffel-goer/pkg/server github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/api github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/events github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/search 
?   	github.com/eiffel-community/eiffel-goer/cmd/goer	[no test files]
ok  	github.com/eiffel-community/eiffel-goer/internal/config	(cached)	coverage: 100.0% of statements
ok  	github.com/eiffel-community/eiffel-goer/internal/database	(cached)	coverage: 100.0% of statements
?   	github.com/eiffel-community/eiffel-goer/internal/database/drivers	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/database/drivers/mongodb	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/logger	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/query	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/requests	[no test files]
ok  	github.com/eiffel-community/eiffel-goer/internal/responses	(cached)	coverage: 100.0% of statements
?   	github.com/eiffel-community/eiffel-goer/internal/schema	[no test files]
ok  	github.com/eiffel-community/eiffel-goer/pkg/application	(cached)	coverage: 100.0% of statements
ok  	github.com/eiffel-community/eiffel-goer/pkg/server	(cached)	coverage: 100.0% of statements
--- FAIL: TestRoutes (0.00s)
    --- FAIL: TestRoutes/EventsReadAll (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8d9857]

goroutine 6 [running]:
testing.tRunner.func1.2({0xc2ae80, 0x13d1670})
	/usr/local/go/src/testing/testing.go:1209 +0x36c
testing.tRunner.func1()
	/usr/local/go/src/testing/testing.go:1212 +0x3b6
panic({0xc2ae80, 0x13d1670})
	/usr/local/go/src/runtime/panic.go:1047 +0x266
github.com/sirupsen/logrus.(*Logger).level(...)
	/home/magnusbk/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:299
github.com/sirupsen/logrus.(*Logger).IsLevelEnabled(...)
	/home/magnusbk/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:321
github.com/sirupsen/logrus.(*Entry).Log(0xc0001f9ce0, 0x2, {0xc000253ae0, 0x1, 0x1})
	/home/magnusbk/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:267 +0x77
github.com/sirupsen/logrus.(*Entry).Error(...)
	/home/magnusbk/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:297
github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/events.(*EventHandler).ReadAll(0xc0001d5740, {0x101b0b0, 0xc000238180}, 0xc000200500)
	/home/magnusbk/src/eiffel-goer/pkg/v1alpha1/handlers/events/events.go:119 +0x3bf
net/http.HandlerFunc.ServeHTTP(0xc0001f47a0, {0x101b0b0, 0xc000238180}, 0x203000)
	/usr/local/go/src/net/http/server.go:2046 +0x4e
github.com/gorilla/mux.(*Router).ServeHTTP(0xc0001e4180, {0x101b0b0, 0xc000238180}, 0xc000200300)
	/home/magnusbk/go/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x367
github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/api_test.TestRoutes.func1(0xc0001d3040)
	/home/magnusbk/src/eiffel-goer/pkg/v1alpha1/api/api_test.go:75 +0x39a
testing.tRunner(0xc0001d3040, 0xc0000710e0)
	/usr/local/go/src/testing/testing.go:1259 +0x230
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1306 +0x727
FAIL	github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/api	0.014s
ok  	github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/events	(cached)	coverage: 25.0% of statements
?   	github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/search	[no test files]
FAIL
```
Pretty sure this is because the tests is creating a *log.Entry with &log.Entry{} which doesn't initialize some fields. Patching myself past that problem I run into this instead:

```
go test -cover -timeout 30s -race github.com/eiffel-community/eiffel-goer/cmd/goer github.com/eiffel-community/eiffel-goer/internal/config github.com/eiffel-community/eiffel-goer/internal/database github.com/eiffel-community/eiffel-goer/internal/database/drivers github.com/eiffel-community/eiffel-goer/internal/database/drivers/mongodb github.com/eiffel-community/eiffel-goer/internal/logger github.com/eiffel-community/eiffel-goer/internal/query github.com/eiffel-community/eiffel-goer/internal/requests github.com/eiffel-community/eiffel-goer/internal/responses github.com/eiffel-community/eiffel-goer/internal/schema github.com/eiffel-community/eiffel-goer/pkg/application github.com/eiffel-community/eiffel-goer/pkg/server github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/api github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/events github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/search 
?   	github.com/eiffel-community/eiffel-goer/cmd/goer	[no test files]
ok  	github.com/eiffel-community/eiffel-goer/internal/config	(cached)	coverage: 100.0% of statements
ok  	github.com/eiffel-community/eiffel-goer/internal/database	(cached)	coverage: 100.0% of statements
?   	github.com/eiffel-community/eiffel-goer/internal/database/drivers	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/database/drivers/mongodb	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/logger	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/query	[no test files]
?   	github.com/eiffel-community/eiffel-goer/internal/requests	[no test files]
ok  	github.com/eiffel-community/eiffel-goer/internal/responses	(cached)	coverage: 100.0% of statements
?   	github.com/eiffel-community/eiffel-goer/internal/schema	[no test files]
ok  	github.com/eiffel-community/eiffel-goer/pkg/application	(cached)	coverage: 100.0% of statements
ok  	github.com/eiffel-community/eiffel-goer/pkg/server	(cached)	coverage: 100.0% of statements
time="2021-11-08T23:36:20+01:00" level=error msg="nofile:1:1 (0): no match found, expected: \"!\", \"bool\", \"double\", \"int\" or [a-zA-Z0-9]" test=true
--- FAIL: TestRoutes (0.01s)
    --- FAIL: TestRoutes/EventsReadAll (0.00s)
        api_test.go:78: Want status '200' for "/v1alpha1/events", got '400'
FAIL
coverage: 100.0% of statements
FAIL	github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/api	0.013s
ok  	github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/events	(cached)	coverage: 25.0% of statements
?   	github.com/eiffel-community/eiffel-goer/pkg/v1alpha1/handlers/search	[no test files]
FAIL
```
It looks like the query parsing code that backs the /events endpoint requires a non-empty query but that's exactly what the test passes.

### Motivation
Tests should pass.

### Exemplification
N/A

### Benefits
Obvious.

### Possible Drawbacks
None.

--- a/pkg/v1alpha1/api/api_test.go
+++ b/pkg/v1alpha1/api/api_test.go
@@ -43,7 +43,7 @@ func TestRoutes(t *testing.T) {
 		statusCode int
 	}{
 		{name: "EventsRead", httpMethod: http.MethodGet, url: "/v1alpha1/events/" + eventID, statusCode: http.StatusOK},
-		{name: "EventsReadAll", httpMethod: http.MethodGet, url: "/v1alpha1/events", statusCode: http.StatusOK},
+		{name: "EventsReadAll", httpMethod: http.MethodGet, url: "/v1alpha1/events?meta.type=EiffelArtifactCreatedEvent", statusCode: http.StatusOK},
 		{name: "SearchRead", httpMethod: http.MethodGet, url: "/v1alpha1/search/" + eventID, statusCode: http.StatusNotImplemented},
 		{name: "SearchUpstreamDownstream", httpMethod: http.MethodPost, url: "/v1alpha1/search/" + eventID, statusCode: http.StatusNotImplemented},
 	}
@@ -62,7 +62,7 @@ func TestRoutes(t *testing.T) {
 	for _, testCase := range tests {
 		t.Run(testCase.name, func(t *testing.T) {
 			ctx := context.Background()
-			app, err := application.Get(ctx, mockCfg, &log.Entry{})
+			app, err := application.Get(ctx, mockCfg, log.NewEntry(log.New()))
 			if err != nil {
 				t.Error(err)
 			}

